.github/workflows/test.yml

```diff
@@ -12,6 +12,9 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
Hey is there any reason you added all these?

This increases the number of github jobs for any PR to 24, which is quite large and can take a long time to complete. It's the most complete way to to test, but unittests themselves and the typescript build does already run on these 3 versions.

Reply:
I dont remember why I did...  perhaps it was a debugging step... I can remove them

Reply:
Dropped the commit